mathml/presentation-markup/fractions/frac-parameters-gap-001-ref.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<meta charset="utf-8">
+<title>Fraction parameters</title>
+<style>
+  math, mspace {
+    font-size: 20px;
```

Comment:
Would it be enough to do it just for math?

Reply:
I think the mspace font-size may change because of math-script-level increment.